src/styles/stars.scss

```
@@ -0,0 +1,32 @@
+.card__stars {
```

and move to `card.scss` file
```suggestion
.card__rating {
```

---

ok